### ISSUE TITLE
[BITV] 9.4.1.2/4.3 - All buttons in the image editor (see screenshot) used for editing an image have not been implemented as such.(1)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41898]
It is best to use native HTML elements such as "button" or the appropriate ARIA role role="button" for implementation. 

If ARIA is used, the element should be made both focusable with tabindex="0" and keyboard-activatable. Additionally, SVG elements should be hidden from screen readers using the "aria-hidden" attribute, and the "aria-selected" attribute should not be used in this case. 

![image](https://github.com/nextcloud/server/assets/6078378/a6d89486-de6e-48e4-b86b-892683034fa1)
![image](https://github.com/nextcloud/server/assets/6078378/6eaef414-b879-4e89-80e2-3d58a88cce36)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v4-n3

**Comment by JuliaKirschenheuter** [comment]
closing as not relevant anymore
